app/models/spree_i18n/translatable.rb

```diff
@@ -1,8 +1,43 @@
 module SpreeI18n
   module Translatable
     extend ActiveSupport::Concern
-    included do
+
+    included do |klass|
       accepts_nested_attributes_for :translations
+      klass.class_eval do
+        class << self
+          alias_method_chain :ransack, :translations
```

Comment:
mostly curious why don't we stick with regular ruby and use `super`? or do we need `alias_method_chain` for some particular reason? we could kill the `klass_eval` here with `super`